tests/functional/pv/pv_services/test_cephfs_pvc_with_necessary_scc_permissions.py

```
@@ -22,7 +22,7 @@ def validate_permissions(pod_obj):
     cmd_output = cmd_output.split()
     assert "root" in cmd_output[4] and cmd_output[13], "Owner is not set to root "
```

Ruff (0.15.15)

[warning] Assertion should be broken down into multiple parts

(PT018)